src/index.html

```diff
@@ -5,8 +5,271 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#Home" class="header__logo Moyo-logo">
+        <img src="./images/logo.png" alt="Moyo-logo" class="Moyo-logo__img">
+      </a>
+      <nav class="header__menu nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#Apple" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#Samsung" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#Phones" class="nav__link">Смартфоны и телефоны</a>
+          </li>
+          <li class="nav__item">
+            <a href="#Pc" class="nav__link" data-qa="nav-hover">Ноутбуки и компьютеры</a>
+          </li>
+          <li class="nav__item">
+            <a href="#Gadgets" class="nav__link">Гаджеты</a>
+          </li>
+          <li class="nav__item">
+            <a href="#Tablets" class="nav__link">Планшеты</a>
+          </li>
+          <li class="nav__item">
+            <a href="#Photo" class="nav__link">Фото</a>
+          </li>
+          <li class="nav__item">
+            <a href="#Video" class="nav__link">Видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <div class="main__content">
+        <div class="product-card" data-qa="card">
+          <div class="product-card__image image">
+            <img src="./images/imac.jpeg" class="image__img" alt="product-card.img">
```

Comment:
```suggestion
            <img 
              src="./images/imac.jpeg"
              class="image__img"
              alt="product-card.img"
            >
```
Если атрибутов больше двух - переноси каждый на новую строку